test/ts-runner.spec.ts

```diff
@@ -130,6 +134,34 @@ test(TsRunner.TSX, async () => {
   expect(syncFn(5)).toBe(5)
 })
 
+test(TsRunner.Node, async () => {
+  const { createSyncFn } = await import('synckit')
+
+  if (nodeVersion < STRIP_TYPES_SUPPORTED_NODE_VERSION) {
+    // eslint-disable-next-line jest/no-conditional-expect
+    expect(() =>
+      createSyncFn<AsyncWorkerFn>(workerMtsPath, {
+        tsRunner: TsRunner.Node,
+      }),
+    ).toThrow('type stripping is not supported in this node version')
+    return
+  }
+
+  let syncFn = createSyncFn<AsyncWorkerFn>(workerJsPath)
```

Comment:
@JounQin I think this may be required, since it will only be the default when running the parent process with the flag enabled, which I don't think we are in the CI.

```suggestion
  let syncFn = createSyncFn<AsyncWorkerFn>(workerJsPath, {
    tsRunner: TsRunner.Node,
  })
```

Reply:
Yeah, conditional setting it would be good, something like `>= 23 ? undefined : TsRunner.Node`.

Reply:
I think the condition should be fine, since the flag is experimental and not too commonly used.